### PR TITLE
Improvements in CSV bundle

### PR DESCRIPTION
**Pull request by rollandf**
- Add RelatedImages field
 - Add Replaces field
 - Add NFD labels information in general description

Signed-off-by: Fred Rolland <frolland@nvidia.com>

### CODE REVIEW

File: config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -64,4 +74,12 @@ spec:
...
     url: https://github.com/Mellanox/network-operator/
+  relatedImages:
+  - image: docker.io/mellanox/mofed-5.5-1.0.3.2@sha256:5376561d6306d7db16860bd6c465c09a2ba2694965826aa043354d29b03f5221
+    name: mofed
+  - image: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin@sha256:16a53286fecdb1e587d3c4c042078974674c3e86c9e98d7dae282f6eb4ee2d8c
+    name: sriov-network-device-plugin
+  - image: nvcr.io/nvidia/cloud-native/k8s-rdma-shared-dev-plugin@sha256:b25c9aa266ba8fd4a21633509b4118664d4b0a52d0d96781011101a90a559414
+    name: rdma-shared-device-plugin
+  replaces: nvidia-network-operator.v1.0.0
   version: 0.0.0
```
Comment (adrianchiris) [inline config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml:85]: version here gets replaced when creating the bundle ?

**Review by adrianchiris — APPROVED** [review]
a few comments, but nothing substantial

File: bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -58,11 +58,21 @@ spec:
     - kind: NicClusterPolicy
       name: nicclusterpolicies.mellanox.com
       version: v1alpha1
-  description: |
-    ## NVIDIA Network Operator
-    The NVIDIA Network Operator simplifies the provisioning and management of NVIDIA networking resources  in a Kubernetes cluster. The operator automatically installs the required host networking software - bringing together all the needed components to provide high-speed network connectivity. These components include the NVIDIA networking driver, Kubernetes device plugin, CNI plugins, IP address management (IPAM) plugin and others.
-    The NVIDIA Network Operator works in conjunction with the NVIDIA GPU Operator to deliver high-throughput, low-latency networking for scale-out, GPU computing clusters.
-    The Network Operator is open-source. For more information on contributions and release artifacts, see the [GitHub repo](https://github.com/Mellanox/network-operator/)
+  description: "## NVIDIA Network Operator\nThe NVIDIA Network Operator simplifies
```
Comment (adrianchiris) [inline bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml:61]: how would you update this ?

isnt it a hassle to add `\n` explicitly ?

using `: |` and just copy pasting what we got from README seems simpler to me.
although further updates are not expected to be large here so its not a big issue

Comment (adrianchiris) [reply bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml:61]: or is that auto generated from README and my comment is irrelevant :) ?

File: config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (rollandf) [reply config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml:85]: This file is the template, so the version will be change when running `make bundle`.

For example:
`DEFAULT_CHANNEL=v1.1.0 CHANNELS=v1.1.0 VERSION=1.1.0 TAG=nvcr.io/nvidia/cloud-native/network-operator@sha256:17afa53f1cf3733c8d0cd282c565975ed5de3124dfc2b7c485ad12c97e51c251 make bundle`

File: bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (rollandf) [reply bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml]: I fixed that. It is taken from the template file :
`config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml`

**Review by e0ne — APPROVED** [review]

**Comment by e0ne** [comment]
CI failure is not related to this change

**Pull request merged**